vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Sequence.java

```diff
@@ -22,18 +26,21 @@
         parameters = @Parameter(name = "prefix", value = "SEQ"), 
         strategy = "edu.asu.diging.vspace.core.data.IdGenerator")
     private String id;
+    
+    @OneToOne(targetEntity = Module.class)
+    private IModule module;
 
-    @OneToMany(targetEntity = Slide.class)
-    private List<ISlide> slides;
-
+    @ManyToMany(targetEntity = Slide.class, cascade=CascadeType.ALL)
```

Comment:
you do not want to delete or save all slides when the sequence is saved/deleted. So this should not be cascade type all.